plugins/producer_plugin/include/eosio/producer_plugin/subjective_billing.hpp

```diff
@@ -59,7 +59,6 @@ class subjective_billing {
    bool                                      _disabled = false;
    trx_cache_index                           _trx_cache_index;
    account_subjective_bill_cache             _account_subjective_bill_cache;
-   block_subjective_bill_cache               _block_subjective_bill_cache;
```

Comment:
Should also remove the `using block_subjective_bill_cache = std::map<account_name, uint64_t>;` 5 lines above.

Reply:
Done. Also made some other minor cleanup changes.